### PR TITLE
new signed readme

### PR DESCRIPTION
**Pull request by KevinSayers**
draft updated readme

Signed-off-by: KevinSayers <sayerskt@gmail.com>

### CODE REVIEW

**Comment by KevinSayers** [comment]
@evanfloden @pditommaso should hybrid workflows across different executors be mentioned? It seems like a very nice feature to me.

File: README.md [hunk 1]
```diff
@@ -1,32 +1,55 @@
...
-without adding unnecessary complexity and letting you concentrate on the flow of data, i.e. the functional logic of the application/algorithm.
-
-It doesn't aim to be another pipeline scripting language yet, but it is built around the idea that the Linux platform 
-is the *lingua franca* of data science, since it provides many simple command line and scripting tools, which by themselves 
-are powerful, but when chained together facilitate complex data manipulations. 
-
-In practice, this means that a Nextflow script is defined by composing  many different processes. 
-Each process can be written in any scripting language that can be executed by the Linux platform (BASH, Perl, Ruby, Python, etc), 
-to which is added the ability to coordinate and synchronize the processes execution by simply specifying their inputs and outputs.   
-
```
[the remark this answers is not in the document]
Comment (KevinSayers) [reply README.md]: Ok sounds good I will add it back in and read over it.

File: README.md [hunk 2]
```diff
@@ -37,185 +60,103 @@ curl -fsSL get.nextflow.io | bash
...
+================
 
-```nextflow
-process perlStuff {
+Containers
+----------------
 
-    """
-    #!/usr/bin/env perl
+*Nextflow* supports both [Docker](https://www.nextflow.io/docs/latest/docker.html) and [Singularity](https://www.nextflow.io/docs/latest/singularity.html) container engines. Additionally, *Nextflow* can easily switch between container engines enabling workflow portability.
```
[the remark this answers is not in the document]
Comment (KevinSayers) [reply README.md]: Added this.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (KevinSayers) [reply README.md]: Made minor changes to it if you want to review

**Comment by pditommaso** [comment]
It looks very good! very minor notes: 

* in the twitter badge I think it should go "@nextflowio" 
* "Modules" -> "Module Environments" otherwise it confuses with programming modules
* "Google Genomics Pipelines API" -> "Google Genomics Pipelines"
* Conda installation: let's put the conda installation command instead of the conda badge: "conda install nextflow -c bioconda" (not sure try it :D)  
* IDE 2017.3 or higher -> "2018.3 or later"

Thanks!

**Comment by KevinSayers** [comment]
@pditommaso editted. Also back to the hybrid workflows, any examples from you or @evanfloden? If not I will make up an example.

**Comment by pditommaso** [comment]
I would go having a smell section mentioning the capability and then linking the example in the [doc](https://www.nextflow.io/docs/latest/awscloud.html#hybrid-workloads). Does it sound good?

**Comment by KevinSayers** [comment]
@pditommaso sounds good. Also regarding the docs, hybrid workflows are not AWS specific. What do you think about putting Hybrid Workflows just under executors?

**Comment by pditommaso** [comment]
I agree!

**Pull request merged**